docs/quick-start/index.md

```diff
@@ -0,0 +1,487 @@
+# MLServer Quick-Start Guide
+
+This guide will help you get started creating machine learning microservices with MLServer 
+in less than 30 minutes. Our use case is to create a service that helps us compare the similarity 
+between two documents. Think about whenever you are comparing a book, news article, blog post, 
+or tutorial to read next, wouldn't it be great to have a way to compare with 
+similar ones that you have already liked (without having to rely on a recommendation's system)? 
+That's what we'll focus on this guide, on creating a document similarity service. 📜 + 📃 = 😎👌🔥
+
+## 00 What is MLServer?
+
+MLServer is an open-source Python library for building production-ready asynchronous APIs for machine learning models.
+
+## 01 Dependencies
+
+The first step is to install `mlserver`, the `spacy` library, and the [language model](https://spacy.io/models/en#en_core_web_lg) 
+`spacy` will need for our use case. We will also download the `wikipedia-api` library to 
+test our use case with a few fun summaries.
+
+If you've never heard of [spaCy](https://spacy.io/) before, it is an open-source Python 
+library for advanced natural language processing that excels at large-scale information 
+extraction and retrieval tasks, among many others. The model we'll use is a pre-trained 
+model on English text from the web. This model will help us get started with our use case 
+faster than if we had to train a model from scratch for our use case.
+
+Let's first install these libraries.
+
+
+```python
+pip install mlserver spacy wikipedia-api
+```
+
+We will also need to download the language model separately once we have spaCy inside our virtual environment.
+
+
+```python
+python -m spacy download en_core_web_lg
+```
+
+If you're going over this tutorial inside a notebook, don't forget to add an exclamation mark `!` 
+in front of the two commands above.
+
+## 02 Set Up
+
+![setup](../assets/mlserver_setup.png)
+
+At its core, MLServer requires that users give it 3 things, a `model-settings.json` file with 
+information about the model, an (optional) `settings.json` file with information related to the server you 
+are about to set up, and a `.py` file with the load-predict recipe for your model (as shown in the 
+picture above).
+
+Let's create a directory for our model.
+
+
+```python
+mkdir -p models_hub/similarity_model
```

Comment:
Slightly nitpicking here but is there any reason we need to create a directory? They could just create the files all in a flat folder structure and then you don't need to append `models_hub/similarity_model` to everything?

Reply:
When I started writing this I intended to create multiple services so the `models_hub` dir made sense. Not needed now so I'll remove it